### PR TITLE
Issue #102, Laravel

### PR DESCRIPTION
**Pull request by pbiswas101**
#102 NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Y

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Y

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

> Yes, I've taken inspiration from various online resources before writing the articles in my own words!

### CODE REVIEW

**Comment by RN0311** [comment]
@pbiswas101  Keep up your spirits high! Good going! :smiley: 
@the-ethan-hunt : Reviewed.

File: guides/DevOps/Laravel.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+**Taylor Otwell** created this elegant framework with it's first beta being released on June 9, 2011.
+
+## Laravel Framework For Web Development
+
+Whether you are a seasoned PHP developer or just starting out; a basic understanding of PHP is a prerequisite for getting acquainted with this particular framework but of course the important question:
+
+> Why should you care?
+
+Well, here's a great article explaining just that -
```
Comment (the-ethan-hunt) [inline guides/DevOps/Laravel.md]: Instead of giving a link, why don't you explain the reasons concisely in the article itself?

**Comment by pbiswas101** [comment]
Okay. Sure.

**Comment by pbiswas101** [comment]
@the-ethan-hunt I've included the necessary changes as requested!
Let me know your thoughts...

**Pull request merged**